### PR TITLE
Makefont improvements

### PR DESCRIPTION
**Pull request by einsteinx2**
I made a few improvements and fixes to `makefont.c` including fixing this issue: https://github.com/rougier/freetype-gl/issues/187

### CODE REVIEW

**Comment by rougier** [comment]
Thanks. Could you add a (small) example to show usage?

**Comment by einsteinx2** [comment]
@einsteinx2 sure, where would you like me to add the example? Here or in the repo somewhere?

**Comment by rougier** [comment]
Not sure since I don't remember why I asked for a small example. Plus there is already one example and your fix should make it work. Do you have a supplementary example in mind or can I merge your fix ?

File: makefont.c [hunk 1]
```diff
@@ -198,16 +198,16 @@ int main( int argc, char **argv )
         {
             ++arg;
 
-            if ( 128.0 != texture_width )
-            {
-                fprintf( stderr, "Multiple --texture parameters.\n" );
-                print_help();
-                exit( 1 );
-            }
+            // if ( 128.0 != texture_width )
```
Comment (adrianbroher) [inline makefont.c]: Please don't comment out sections. Delete them if they are not needed.

Comment (einsteinx2) [reply makefont.c]: Totally agree, not sure why I did that (was probably while I was trying things out).

Comment (einsteinx2) [reply makefont.c]: Done.

**Comment by einsteinx2** [comment]
@rougier nope I didn't have anything in mind, you can go ahead and merge it.

**Comment by rougier** [comment]
Thanks ! Merged !

**Pull request merged**